src/constants/apiDedupe.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+angular.module('openshiftCommonServices')
+  .constant('API_DEDUPLICATION', {
+    // Exclude duplicate kinds we know about that map to the same storage as another
+    //  group/kind.  This is unusual, so we are special casing these.
+    groups: [{group: 'authorization.openshift.io'}],
+    kinds: [
+      {group: 'extensions', kind: 'HorizontalPodAutoscaler'},
+      {group: 'apps', kind: 'DaemonSet'}
```

Comment:
The apps group is the one we want to keep. The extensions group is the one to filter.

Reply:
got it.